### ISSUE TITLE
Upgrading from Twisted 10.1 to trunk HEAD (after 11.0) results in many TLSBioFactory log calls

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5073]
|<img alt="rlotun's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| rlotun reported|
|-|-|
|Trac ID|trac#5073|
|Type|enhancement|
|Created|2011-04-28 13:59:38Z|

I just picked one server from our cluster to upgrade to twisted trunk HEAD (r31733) (mainly to see what effect [#5011](https://github.com/twisted/twisted/issues/5011) had on our server), resulted in many more log lines that looked like this:

```
2011-04-28 13:17:29+0000 [TLSMemoryBIOProtocol,client] Stopping factory <twisted.protocols.tls.TLSMemoryBIOFactory instance at 0xa00530c>
2011-04-28 13:17:29+0000 [TLSMemoryBIOProtocol,client] Stopping factory <twisted.protocols.tls.TLSMemoryBIOFactory instance at 0x9ffe30c>
2011-04-28 13:17:29+0000 [TLSMemoryBIOProtocol,client] Stopping factory <twisted.protocols.tls.TLSMemoryBIOFactory instance at 0xa0b348c>
```

I temporarily turned this off by putting a `noisy = False` at [twisted/protocols/tls.py@31733#L337](twisted/protocols/tls.py@31733#L337) but I'm not sure how best to tackle the general case. Shouldn't this factory respect the noisy flag of my `twisted.web.client.HTTPClientFactory`?

<details><summary>Searchable metadata</summary>

```
trac-id__5073 5073
type__enhancement enhancement
reporter__rlotun rlotun
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__duplicate duplicate
component__core core
keywords__ 
time__1303999178000000 1303999178000000
changetime__1318801709000000 1318801709000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

See [#5062](https://github.com/twisted/twisted/issues/5062) for a similar (not duplicate) issue.

**Comment by twisted-trac** [comment]
|<img alt="rlotun's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|rlotun set status to `closed`|
|-|-|

This was addressed by a duplicate ticket: [#5311](https://github.com/twisted/twisted/issues/5311)
